reference/docs-conceptual/community/contributing/file-an-issue.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How to file an issue
+description: This article explains how to give feedback about the PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# How to file an issue
+
+There are two ways to create an issue:
+
+1. Use the feedback controls at the bottom of the page.
+1. File an issue in GitHub directly
+
+## Using the feedback controls
+
+For a full description of the feedback controls, see the Docs Team blog announcing this [feature][feedback].
+
+At the bottom of most pages on `docs.microsoft.com`, you'll see two feedback buttons. One is a link
+for product feedback and one is for documentation feedback. The documentation feedback requires a
+GitHub account. Clicking the button authenticates you in GitHub, then presents a simple form to
+enter your feedback. When you submit the form, the feedback control creates a new GitHub issue and
+links it to the current article.
+
+> [!NOTE]
+> This is not a support channel. This is a way to ask clarifying questions about documentation or
+> report errors and omissions. If you need technical support, see the
```

Comment:
I'd rephrase to just `see [Community resources](../community-support.md).`

Reply:
Fixed